pkgs/hooks_runner/test/build_runner/build_planner_test.dart

```diff
@@ -25,6 +25,7 @@ void main() async {
         const LocalFileSystem(),
         nativeAddUri,
         'native_add',
+        false,
```

Comment:
Maybe upgrade `includeDevDependencies` to a named parameter everywhere to make this more readable? 

See also https://dart.dev/effective-dart/design#avoid-positional-boolean-parameters.

There's also a lint for this: `avoid_positional_boolean_parameters`